### PR TITLE
Create links for speakers in admin/reports#index

### PR DESCRIPTION
**Pull request by selini**
Speaker names did not have links.We create the links in the admin namespace

### CODE REVIEW

File: app/views/admin/reports/_all_events.html.haml [hunk 1]
```diff
@@ -27,7 +27,13 @@
           %td
             = link_to event.title, edit_admin_conference_program_event_path(@conference.short_title, event)
             %br
-            .small (Presented by #{event.speaker_names})
+            .small
+              (Presented by
+              - speakers = event.speakers.uniq
```
Comment (Ana06) [inline app/views/admin/reports/_all_events.html.haml]: why is the `uniq` needed?

File: app/views/admin/reports/_all_events.html.haml [hunk 2]
```diff
@@ -27,7 +27,13 @@
           %td
             = link_to event.title, edit_admin_conference_program_event_path(@conference.short_title, event)
             %br
-            .small (Presented by #{event.speaker_names})
+            .small
+              (Presented by
```
Comment (ChrisBr) [inline app/views/admin/reports/_all_events.html.haml]: https://de.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself

As you use this three times, maybe you should move this code into a helper or partial?
Also please have a look into ``join``

``
["one", "two"].join(", ")
-> "one, two"
``

**Comment by ChrisBr** [comment]
@selini otherwise very nice :+1: 
Am I right that you apply for Rails Girls?

File: app/views/admin/reports/_all_events.html.haml [hunk 3]
```diff
@@ -27,7 +27,13 @@
...
+              - speakers = event.speakers.uniq
+              - speakers.each.with_index(1) do |speaker,index|
```
Comment (Ana06) [inline app/views/admin/reports/_all_events.html.haml]: what about writing this like:

``` Ruby
event.speakers.map{ |speaker| link_to speaker.name, admin_user_path(speaker) }.join(', ').html_safe
```

**Comment by selini** [comment]
I have completed the changes you have requested

**Comment by differentreality** [comment]
Nice work! :+1: Please squash your commits for merging.

File: app/views/admin/reports/_all_events.html.haml [hunk 4]
```diff
@@ -27,7 +27,10 @@
           %td
             = link_to event.title, edit_admin_conference_program_event_path(@conference.short_title, event)
             %br
-            .small (Presented by #{event.speaker_names})
+            .small
+              (Presented by
+              =speaker_links(event)
```
Comment (differentreality) [inline app/views/admin/reports/_all_events.html.haml]: space after =

**Comment by differentreality** [comment]
:+1: Thanks @selini, good work! :tada:

**Pull request merged**